pypeit/ginga/PypeIt.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
```

Comment:
Add a short comment explaining the purpose of the module.

Reply:
Done.